### ISSUE TITLE
Link colour in settings sidebar doesn't meet AA contrast

### ISSUE THREAD

**Issue by Stanton** [jadu/pulsar#997]
Links within the settings sidebar don't meet WCAG 2.1 AA contrast.

![Screenshot 2019-06-24 at 14 38 11](https://user-images.githubusercontent.com/18653/60023548-ea7b1200-968d-11e9-95d2-ba19ee5068e0.png)
![Screenshot 2019-06-24 at 14 39 39](https://user-images.githubusercontent.com/18653/60023553-ec44d580-968d-11e9-992d-3c79b1af0685.png)

**Comment by Stanton** [comment]
Self-inflicted by not using a href attribute in prototype code 🤦‍♂
